### PR TITLE
Abstract configuration wizard into package

### LINKED ISSUE #35: Abstract configuration-wizard into package
## Summary
We need a new package for the configuration-wizard: `@yoast/configuration-wizard`.

## Explanation
The contents of this composite should be included in this new package:
- yoast-components/composites/OnboardingWizard

yoast-components should still export this composite if they did before, so we don't break backwards compatibility.
All usages of the composites inside the javascript repository should be updated. They should use the new `@yoast/configuration-wizard` package.

### PR DESCRIPTION
**Pull request by bintzandt**
Fixes #35 

# Summary
Abstract the configuration wizard into a new package.

### CODE REVIEW

File: packages/configuration-wizard/onboarding-wizard.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
-@import "../../style-guide/colors";
+//TODO: Remove this reference.
```
Comment (diedexx) [inline packages/configuration-wizard/onboarding-wizard.scss]: comment

File: packages/configuration-wizard/package.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    "testURL": "http://localhost",
+    "testRegex": ".*Test.js$",
+    "setupTestFrameworkScriptFile": "<rootDir>/jest/setupTests.js",
+    "collectCoverageFrom": [
+      "**/*.js",
+      "!**/*.*Test.js",
+      "!grunt/**/*.js",
+      "!*.js",
+      "!app/**/*.js",
+      "!a11y/examples/example.js",
```
Comment (diedexx) [inline packages/configuration-wizard/package.json]: are these references still valid?

File: packages/configuration-wizard/onboarding-wizard.scss (hunk 1 above)
Comment (bintzandt) [reply packages/configuration-wizard/onboarding-wizard.scss]: I'm not sure how to rewrite the import. Could you have a look with me?

File: packages/configuration-wizard/package.json (hunk 2 above)
Comment (bintzandt) [reply packages/configuration-wizard/package.json]: Nope, they can be removed.

File: packages/configuration-wizard/onboarding-wizard.scss (hunk 1 above)
Comment (bintzandt) [reply packages/configuration-wizard/onboarding-wizard.scss]: Or tomorrow 👋

**Comment by diedexx** [comment]
ACC 👍 
Waiting for the build

**Pull request merged**